test/generate-inputs/test_generate_inputs.py

```diff
@@ -49,3 +49,6 @@ def test_prepare_inputs_networks(self):
             exp_file_name = algo_exp_file[algo]
             assert filecmp.cmp(OUTDIR + f"{algo}-{exp_file_name}.txt", EXPDIR + f"{algo}-{exp_file_name}-expected.txt",
```

Comment:
Would this fail along with the new assert in 53-54 adds? Or was this blindly passing?

Reply:
This would not fail, since we only check against one of the required input files.

Reply:
(Though, generally, we do have a deep weakness against all of these tests, which is that we do no full workflow runs other than the Snakemake run in CI. That run carries a lot of testing weight with it that it really should not have.)